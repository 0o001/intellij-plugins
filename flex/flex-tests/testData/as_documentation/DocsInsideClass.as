package {
public class DocsIn<caret>sideClass {

    /**
     *  this is docs for the function
     */
      public function foo() {}


}
}