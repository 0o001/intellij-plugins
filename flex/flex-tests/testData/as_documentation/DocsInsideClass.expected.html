<div class='definition'><pre>DocsInsideClass</pre></div>